crates/bevy_state/src/state_scoped.rs

```diff
@@ -16,8 +16,7 @@ use crate::state::{StateTransitionEvent, States};
 /// Entities marked with this component will be removed
 /// when the world's state of the matching type no longer matches the supplied value.
 ///
-/// To enable this feature remember to configure your application
-/// with [`enable_state_scoped_entities`](crate::app::AppExtStates::enable_state_scoped_entities) on your state(s) of choice.
+/// To enable this feature remember to add the attribute `#[states(scoped_entities)]` when deriving [`States`].
```

Comment:
This could still refer to `enable_state_scoped_entities` as an alternative to the macro attribute